Source/WebCore/Modules/url-pattern/URLPatternParser.cpp

```diff
@@ -508,7 +508,7 @@ String generatePatternString(const Vector<Part>& partList, const URLPatternStrin
 template<typename CharacterType>
 static String escapePatternStringForCharacters(std::span<const CharacterType> characters)
 {
-    static constexpr std::array escapeCharacters { '+', '*', '?', ':', '(', ')', '\\', '{', '}' }; // NOLINT
+    static constinit std::array<CharacterType, 9> escapeCharacters { '+', '*', '?', ':', '(', ')', '\\', '{', '}' }; // NOLINT
```

Comment:
This should be changed back from `constinit`, which I misunderstood as better than `constexpr`:
```suggestion
    static constexpr std::array<CharacterType, 9> escapeCharacters { '+', '*', '?', ':', '(', ')', '\\', '{', '}' }; // NOLINT
```

Reply:
You could also consider removing the need for the explicit count by using std::to_array

```c++
static constexpr auto escapeCharacters = std::to_arrat<CharacterType>({ '+', '*', '?', ':', '(', ')', '\\', '{', '}' });
```